test/Microsoft.Azure.WebJobs.Host.UnitTests/WebJobs.Host.UnitTests.csproj

```diff
@@ -18,6 +18,11 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <WarningsAsErrors />
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="Singleton\**" />
```

Comment:
Why did we add this?

Reply:
It was an empty folder. Just cleaning it up :)